packages/grid/stories/grid.doc.stories.mdx

```diff
@@ -148,24 +169,60 @@ This grid also shows an example of a custom cell displaying a basic chart. Switc
 
 ### Editable cells
 
-Grid allows you to make cell values editable in place.
-Use the `CellEditor` component as a child of `GridColumn` to make the column editable.
-`CellEditor` is a non-rendered component.
-`CellEditor` is expected to have one child representing the specific editor component to use for the column, e.g. `TextCellEditor`, `DropdownCellEditor` or `NumericCellEditor`.
+The grid allows you to make cell values editable in place.
+
+Use the `CellEditor` (a non-rendered) component as a child of `GridColumn` to make the column editable.
+`CellEditor` is expected to have one child representing the specific editor component to use for the column.
 The child of the `CellEditor` component gets rendered in the focused cell when the grid goes to edit mode.
 
+We have [three sample components](https://uitk.pages.dev/?path=/story/grid-new-grid--editable-cells) created for use as `CellEditor`'s child,
+i.e. `TextCellEditor`, `DropdownCellEditor` and `NumericCellEditor`.
```

Comment:
Let's remove DropdownCellEditor as it is not fully supported yet.

Reply:
yup it's been removed